include/model/create_document.js

```diff
@@ -106,18 +105,6 @@ module.exports = function DocumentCreatorModule(pb) {
     };
 
     /**
-     * Formats usernames
-     *
-     * @method usernameFormatting
-     * @param {Object} post Key value pair object
-     */
-    DocumentCreator.usernameFormatting = function(post){
-        if(util.isString(post['username'])) {
-            post['username'] = post['username'].toLowerCase();
-        }
-    };
```

Comment:
I guess I don't understand why the call needed to be removed from the create function